include/nfc/tnep/poller.rst

```diff
@@ -13,12 +15,20 @@ The NFC Tag Device supports all tag types from Type 2 Tag to Type 5 Tag.
 The Poller communicates with the type of tag corresponding to the NFC Tag Device type.
 The communication occurs through Poller's Reader/Writer role.
 
+.. tnep_intro_end
+
+Initialization
```

Comment:
@KAGA164 , as discussed on Teams, please update this section when you have time.